tools/wbuilder.lua

```diff
@@ -460,5 +461,6 @@ UIManager:show(Clock:new())
 -- testKeyValuePage()
 -- testTouchProbe()
 -- testBookStatus()
-testNetworkSetting()
+--testNetworkSetting()
+UIManager:show(DoubleSpinWidget:new{})
```

Comment:
Should these changes to wbuilder (that I never used) stay in the commits?

Reply:
Occasionally yes, but not these. I think I may have accidentally overlooked some changes earlier unless this snuck in after rebasing.

Reply:
With my understanding, wbuilder looks like a place to quickly test widget without start whole reader. I changed here to debug some bugs quickly. 

wbuilder changes are not necessary.

Reply:
Of course, it's there to be used. It's however almost never to be committed. ^_^